omero/sysadmins/admins-with-restricted-privileges.rst

```diff
@@ -17,9 +17,11 @@ facility managers, image analysts, or anybody who needs to organize
 users and data of others in OMERO.
 
 Full administrators in OMERO can create new administrators with
-restricted privileges using the OMERO.web
-interface, see the :help:`create new users <sharing-data#admin>` section
-of our Help documentation.
+restricted privileges using the OMERO.web interface, see the
+:help:`create new users <sharing-data#admin>` section of our Help
+documentation. With considerably more difficulty one may also :doc:`use
```

Comment:
Do we want to rephrase this to say it is possible but not recommended? Maybe:
"Support for creating restricted administrators is yet to be added to OMERO.cli. It is possible to [use OMERO.cli to adjust restrictions] on an existing administrator but we do not recommend this as the [permissions mapping] may cause confusion."